transports/websocket-websys/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Unreleased
```

Comment:
```suggestion
## 0.3.2
```

Also needs a version bump in `Cargo.toml` and `transports/websocket-websys/Cargo.toml`

Reply:
fixed in [fbf4a49](https://github.com/libp2p/rust-libp2p/pull/5025/commits/fbf4a49e6476a39b7e02a380b1251d58deb48abb)